### DISCUSSION TITLE
Connect multiple wallets at same time

### DISCUSSION THREAD

**Discussion by SwapnilSoni1999** [reown-com/appkit#3320] [General]
Hi I'm lookoing for a way to connect more than 1 wallet at a time
for eg. I have a ethereum and a solana wallet to be used in website to interact with.
How do I achieve this functionality.

**Comment by arein** [comment]
@SwapnilSoni1999 can you explain the use-case more? Do all of yours users need to have both an EVM and an SVM wallet connected to proceed?

**Reply by SwapnilSoni1999** [reply]
yes exactly 
I have a project where users have to connect those two wallets in order to proceed with transfer of NFTs from one network to another and to verify signatures from both wallets at same time for full transaction proceeding

**Comment by ethan-auroria** [comment]
I would like to bump this up, as I have an use case where we have multiple NFT collections on both Ethereum and Solana.
The appropriate way to achieve this is by making AppKit not using global data storage, or at least the storage is separated and tagged with some kind of unique ID, thus allow for multiple instances of AppKit running side by side.
